packages/teams-js/src/public/location.ts

```diff
@@ -77,6 +78,9 @@ export namespace location {
       if (!props) {
         throw { errorCode: ErrorCode.INVALID_ARGUMENTS };
       }
+      if (!isSupported()) {
+        throw new Error(errorNotSupportedOnPlatform);
```

Comment:
We should be throwing SdkError not Error. This should apply to all places in the code where we check `isSupported`.

Reply:
Okay, I'll make those changes everywhere.

Reply:
Resolved in the latest commit.